### DISCUSSION TITLE
Make it more usable for Python

### DISCUSSION THREAD

**Discussion by jk-85** [tsujan/FeatherPad#836] [Ideas]
I know it's annoying for you, and I don't know why it's such a big deal and don't follow all other (older) conversations....... but:

Seems it's a C/CPP only editor or in general one for all languages except Python that has no coding brackets:
You have to press CTRL+TAB every time to get spaces instead of tabs, unless you consistently use TABs always and strictly avoid to use spaces. Then, and only then it may work, but then you absolutely have to be careful to NOT accidently click even one time on Edit->"Text Tabs to Spaces". Then your entire work has spaces only and you never can roll back in your code.

**Comment by tsujan** [comment]
I've written many Python scripts with FeatherPad.

Any app may reflect what its developer prefers. I didn't like other editors and so, made mine. One of the things that I disliked in almost all editors was that soft and hard tabs were a matter of either-or. Of course, I wouldn't want that for FeatherPad.

Feature requests are always appreciated, but that doesn't mean they will always be implemented. This program and its source are shared for free. Some people may like it, some not; that's understandable but not interesting to me.

So, please don't repeat your comments and arguments; they were read the first time.

**Comment by jk-85** [comment]
Yeah, thanks. Will not write more about that topic :-)

**Comment by Laurencio** [comment]
Sorry, don't want to bother.

Does this mean you would avoid to consider an option to "use Tabs as spaces", to write spaces instead of tabs when pressing the Tab key, like other editors have, in featherpad? I was going to ask for that as a feature request!

As a workaround i' ve been using the Text tabs to spaces after some writing (with a shortcut) in .py scripts when i get a "inconsisent use of tabs and spaces" error message.
I found out about the "CTRL + TAB" shortcut when I read this thread, I didn't know it existed. I didn't saw it in the customizable shorcuts list tab . So i think they can not be modified, like interchanging them: Tab to write spaces and Ctrl + Tab to write tabs...

By the way, I like featherpad very much. I do try other editors but always get back to featherpad.
Greetings

**Reply by tsujan** [reply]
Help → Text Tabbing
